src/Module.php

```diff
@@ -108,4 +118,9 @@ public function onAuthenticationPost(MvcAuthEvent $e)
 
         $this->container->setService('api-identity', $e->getIdentity());
     }
+
+    public function getMvcRouteListener()
```

Comment:
Why did you add this method? It was never part of the class previously, and I don't see it used in the unit tests...